client/components/token-field/test/index.jsx

```diff
@@ -32,89 +35,81 @@ const charCodes = {
+		div.innerHTML = node.html();
+		return div.firstChild.innerHTML;
+	}
+
 	function getTokensHTML() {
-		var textNodes = tokenFieldNode.querySelectorAll( '.token-field__token-text' );
-		return Array.prototype.slice.call( textNodes ).map( function( el ) {
-			return el.innerHTML;
-		} );
+		const textNodes = tokenFieldNode.find( '.token-field__token-text' );
+
+		return textNodes.map( getNodeInnerHtml );
 	}
 
 	function getSuggestionsHTML( selector ) {
-		var suggestionNodes = tokenFieldNode.querySelectorAll( selector || '.token-field__suggestion' );
-		return Array.prototype.slice.call( suggestionNodes ).map( getSuggestionNodeHTML );
+		const suggestionNodes = tokenFieldNode.find( selector || '.token-field__suggestion' );
+
+		return suggestionNodes.map( getSuggestionNodeHTML );
 	}
 
 	function getSuggestionNodeHTML( node ) {
-		var suggestionContent = node.childNodes[0];
-		if ( suggestionContent.nodeName === '#text' ) {
-			// This suggestion is not part of a partial match; just return
-			// the whole suggestion
-			return node.innerHTML;
+		if ( ! node.find( 'span' ).length ) {
+			return getNodeInnerHtml( node );
 		}
 
 		// This suggestion is part of a partial match; return the three
 		// sections of the suggestion (before match, match, and after
 		// match)
-		return Array.prototype.slice.call( suggestionContent.childNodes ).map( function( child ) {
-			return child.innerHTML;
-		} );
+		const div = document.createElement( 'div' );
+		div.innerHTML = node.find( 'span' ).html();
+
+		return map( div.firstChild.childNodes, childNode => childNode.innerHTML );
 	}
 
 	function getSelectedSuggestion() {
 		var selectedSuggestions = getSuggestionsHTML( '.token-field__suggestion.is-selected' );
-		return selectedSuggestions[0] || null;
+
+		return selectedSuggestions[ 0 ] || null;
 	}
 
-	useFakeDom.withContainer();
+	useFakeDom();
 
 	useMockery( mockery => {
-		ReactDom = require( 'react-dom' );
-		React = require( 'react' );
-		TestUtils = require( 'react-addons-test-utils' );
-
-		EMPTY_COMPONENT = require( 'test/helpers/react/empty-component' );
-
-		mockery.registerMock( 'components/tooltip', EMPTY_COMPONENT );
+		mount = require( 'enzyme' ).mount;
+		mockery.registerMock( 'components/tooltip', EmptyComponent );
 		TokenFieldWrapper = require( './lib/token-field-wrapper' );
 	} );
 
-	before( function() {
-		reactContainer = useFakeDom.getContainer();
-	} );
-
 	beforeEach( function() {
-		wrapper = ReactDom.render( <TokenFieldWrapper />, reactContainer );
-		tokenFieldNode = ReactDom.findDOMNode( wrapper.refs.tokenField );
-		textInputNode = tokenFieldNode.querySelector( '.token-field__input' );
-		TestUtils.Simulate.focus( textInputNode );
-	} );
-
-	afterEach( function() {
-		ReactDom.unmountComponentAtNode( reactContainer );
+		wrapper = mount( <TokenFieldWrapper /> );
+		tokenFieldNode = wrapper.ref( 'tokenField' );
+		textInputNode = tokenFieldNode.find( '.token-field__input' );
+		textInputNode.simulate( 'focus' )
```

Comment:
Nitpick, but also to raise an aside: Should be a semi-colon here, and I'm at a loss for why the [`semi` rule](http://eslint.org/docs/rules/semi) isn't enabled in our ESLint configuration. At some point, we may have upgraded to an ESLint version where default rules were not specified, but it's one we should be enforcing ([related guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#semicolons)).